posts/month-in-mathlib-nov-2021.md

```diff
@@ -0,0 +1,84 @@
+This sequence of PRs shows the main theorem on differentiation of measures: given two measures `ρ` and `μ` on a finite-dimensional real vector space, the ratio `ρ (ball x r) / μ (ball x r)` converges `μ`-almost everywhere to the Radon-Nikodym derivative of `ρ` with respect to `μ` when `r` tends to `0`. The theorem is in fact proved in greater generality (Vitali families, as in Federer's famous book on geometric measure theory).
+
+* Analysis.
+  - [PR #10145](https://github.com/leanprover-community/mathlib/pull/10145) Introduce `C*`-algebras
+  - [PR #10390](https://github.com/leanprover-community/mathlib/pull/10390) Define spectrum and prove basic prope...
+  - [PR #10530](https://github.com/leanprover-community/mathlib/pull/10530) Prove properties of spectrum in a Banach algebra
+  - [PR #9840](https://github.com/leanprover-community/mathlib/pull/9840) Rayleigh quotient produces eigenvectors
+  - [PR #9995](https://github.com/leanprover-community/mathlib/pull/9995) diagonalization of self-adjoint endomorphisms (finite dimension)
+  - [PR #10317](https://github.com/leanprover-community/mathlib/pull/10317) more concrete diagonalization theorem
+  - [PR #10335](https://github.com/leanprover-community/mathlib/pull/10335) Continuous affine maps are smooth
+
+* Convergence.
+  - [PR #10073](https://github.com/leanprover-community/mathlib/pull/10073) Convergence of a sequence which does not oscillate infinitely
+  - [PR #10258](https://github.com/leanprover-community/mathlib/pull/10258) Prove that, if u_n is subadditive, then u_n / n converges.
+
+* Seminorms
+  - [PR #9097](https://github.com/leanprover-community/mathlib/pull/9097) Define the Minkowski functional
+  - [PR #10203](https://github.com/leanprover-community/mathlib/pull/10203) A seminorm is convex
+
+* Homotopy theory.
+  - [PR #9865](https://github.com/leanprover-community/mathlib/pull/9865) Add `homotopic` for `continuous_map`s.
+  - [PR #10195](https://github.com/leanprover-community/mathlib/pull/10195) The functor from `Top` to `Groupoid`
+
+* Number theory.
+  - [PR #9071](https://github.com/leanprover-community/mathlib/pull/9071) Define the class number of number fields and function fields
+  This finishes the finiteness proof of the class group of a number field and function field, described in [a previous blog post](https://leanprover-community.github.io/blog/posts/dedekind-domains-and-class-number-in-lean/).
+  - [PR #8820](https://github.com/leanprover-community/mathlib/pull/8820) Add theorem for Lucas primality test
+
+* Group theory.
+  - [PR #10059](https://github.com/leanprover-community/mathlib/pull/10059) (Infinite) Sylow subgroups are isomorphic
+  - [PR #10283](https://github.com/leanprover-community/mathlib/pull/10283) Prove the full Schur-Zassenhaus theorem
+  - [PR #10249](https://github.com/leanprover-community/mathlib/pull/10249) Define the exponent of a group
+
+* Sheafification.
```

Comment:
It may be worthwhile to mention that these PRs work with with a general Grothendieck topology, as opposed to a topological space (where sheafification of type-valued presheaves has existed for some time now).

Reply:
Done